pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/con_pantheon_yaml_file.adoc

```diff
@@ -17,17 +16,51 @@ The configuration file enables you to import content from a Git repository into
 The {ProductShortName} configuration file defines the following parameters:
 
 * `server` The URL of the {ProductShortName} server.
-* `repositories` A multi-valued structure with the following child members:
+* `repository` The name of the repository. It is recommended not to include spaces in the repository name.
```

Comment:
Hey @emarcusRH, the repository attribute doesn't equal the repository name. When writers want to import new version of their docs they need to change the repository attribute. 
![image](https://user-images.githubusercontent.com/58083949/137879175-220f7bab-dc40-4771-9acc-7cd10341d1d6.png)
I think this information that I got from Andrew might be useful.